.github/workflows/pythonpackage.yml

```diff
@@ -5,7 +5,7 @@ name: Python package
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, userconfig ]
```

Comment:
Why did you add the useconfig branch to the build bot workflow file? Once I merge this PR, the userconfig branch will be deleted automatically. For me this looks like an error.

Reply:
jup, this was just for testing